src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -376,6 +379,7 @@ function TextArea({
           open={popoverOpen}
           onClose={() => setPopoverOpen(false)}
           anchorElementRef={caretElementRef}
+          loadingUsers={usersForMentions.status === 'loading'}
```

Comment:
This is fine, though alternatively you could make the `users` prop a `UsersForMentions` value as well, preserving the `state` value if loading and passing a filtered list if the state is `loaded`.